pkg/machine/qemu/machine.go

```diff
@@ -30,6 +30,13 @@ var (
 	vmtype = "qemu"
 )
 
+const (
+	VolumeTypeVirtfs  = "virtfs"
+	VolumeTypeSshfs   = "sshfs"
+	MountType9p       = "9p"
+	MountTypeSSHocker = "sshocker"
+)
```

Comment:
Why not call this "sshfs" (or "reverse-sshfs")

Reply:
I agree lets call it sshfs

Reply:
It's a reverse sshfs mount over a ssh slave connection, but will change to "sshfs".